docs/run-tests-locally.md

```diff
@@ -37,59 +26,26 @@ Accessibility and UI tests are written using [Cypress](https://www.cypress.io/),
 
     # install dependencies
     npm install
-
-    # run tests 
-    npm run cy:run # run tests in headless mode
-    npm run cy:open # run tests in headed mode
-
     ```
 
-If you need to update the mocked test data for UI and Accessibility tests see [update test data](./update-test-data.md)
-
-### Security testing with ZAP
-
-The Cypress tests can also be run, proxied via [OWASP ZAP](https://zaproxy.org) for passive security scanning of the application.
-
-These can be run using the configured `docker-compose.yml`, which will spin up containers for the ZAP daemon and the Cypress tests, including all networking required. You will need to update any config in the file before running
-
-Create a `.env` file for docker, this file needs to include
-
-- all of your required cypress configuration
-- HTTP_PROXY e.g. `http://zap:8080`
-- ZAP_API_KEY, can be any random guid
-
-Example env:
-
-```env
-URL=<Enter URL>
-USERNAME=<Enter username>
-API=<Enter API>
-API_KEY=<Enter API key>
-AUTH_KEY=<Enter auth key>
-HTTP_PROXY=http://zap:8080
-ZAP_API_KEY=<Enter random guid>
+2. DFE machines may have trouble running Cypress for the first time or installing dependancies due to internal proxies and auth. If in the instance this looks like its occuring on setup you will need to run the following command to bypass this for the installation.
 
-```
-
-**Note**: You might have trouble running this locally because of docker thinking localhost is the container and not your machine
-
-To run docker compose use:
-
-`docker-compose -f docker-compose.yml --exit-code-from cypress`
-
-**Note**: `--exit-code-from cypress` tells the container to quit when cypress finishes
+    ```bash
+    export NODE_TLS_REJECT_UNAUTHORIZED=0'
```

Comment:
We shouldn't be recommending this method - I'm writing some docs about a secure way of getting Cypress to work on a DfE machine. 
I think we should replace this with a prompt to look in internal DfE docs for help getting the laptop set up